cmd/tendermint/commands/reindex_event_test.go

```diff
@@ -89,8 +89,8 @@ func TestLoadEventSink(t *testing.T) {
 		{[]string{"NULL"}, "", true},
 		{[]string{"KV"}, "", false},
 		{[]string{"KV", "KV"}, "", true},
-		{[]string{"PSQL"}, "", true},         // true because empty connect url
-		{[]string{"PSQL"}, "wrongUrl", true}, // true because wrong connect url
+		{[]string{"PSQL"}, "", true},          // true because empty connect url
+		{[]string{"PSQL"}, "wrongUrl", false}, // true because wrong connect url
```

Comment:
```suggestion
		{[]string{"PSQL"}, "", true},         // true because empty connect url
		{[]string{"PSQL"}, "wrongUrl", true}, // true because wrong connect url
```
revert